src/UserService.Data.Provider.MsSql.Ef/Migrations/20210317164900_AddDbSkills.cs

```diff
@@ -0,0 +1,71 @@
+﻿using LT.DigitalOffice.UserService.Models.Db;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+
+namespace LT.DigitalOffice.UserService.Data.Provider.MsSql.Ef.Migrations
+{
+    [DbContext(typeof(UserServiceDbContext))]
+    [Migration("20210317164900_AddDbSkills")]
+    class AddDbSkills : Migration
+    {
+        private void CreateSkillsTable(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: DbSkill.TableName,
+                columns: table => new
+                {
+                    Id = table.Column<Guid>(nullable: false),
+                    SkillName = table.Column<string>(nullable: false)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_Skills", x => x.Id);
+                });
+        }
+
+        private void CreateUsersSkillsTable(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: DbUserSkills.TableName,
+                columns: table => new
+                {
+                    Id = table.Column<Guid>(nullable: false),
+                    UserId = table.Column<Guid>(nullable: false),
+                    SkillId = table.Column<Guid>(nullable: false)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_UserSkills", x => x.Id);
+
+                    table.ForeignKey(
+                        name: "FK_UserSkills_Users",
+                        column: x => x.UserId,
+                        principalTable: DbUser.TableName,
+                        principalColumn: "Id",
```

Comment:
use nameof(DbUserSkills.Id) and make also for other

Reply:
fixed